helm-charts/bifrost/README.md

```
@@ -0,0 +1,399 @@
+```
+
+## Accessing Bifrost
+
+### Port Forward (ClusterIP)
+
+```bash
+export POD_NAME=$(kubectl get pods -l "app.kubernetes.io/name=bifrost,app.kubernetes.io/instance=bifrost" -o jsonpath="{.items[0].metadata.name}")
+kubectl port-forward $POD_NAME 8080:8080
+```
+
+Then access at http://localhost:8080
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)